src/topology/metric_space/hausdorff_distance.lean

```diff
@@ -433,6 +444,17 @@ begin
     { simp [ennreal.add_eq_top, inf_edist_ne_top hs, edist_ne_top] }}
 end
 
+lemma disjoint_closed_ball_of_lt_inf_edist {r : ℝ} (h : r < inf_dist x s) :
+  disjoint (closed_ball x r) s :=
```

Comment:
is `disjoint_closed_ball_of_lt_inf_edist` not useful to prove this?

Reply:
Not really because the map from `ennreal` to `real` sends top to `0`, and the map from `real` to `ennreal` sends negative numbers to zero. It is faster to just copy the proof than to try to juggle with these issues.